### ISSUE TITLE
TypeError: can't assign to property "lang" on false: not an objectapp.js

### ISSUE THREAD

**Issue by valasek** [patrickmonteiro/quasar-speech-api#2]
Hi Patrick, thanks for sharing this example.
When I run quasar dev on localhost on the repo and hit Record I am getting "return error". What am I doing wrong? Any hint appreciated.

```
return error Index.vue:107
TypeError: can't assign to property "lang" on false: not an objectapp.js line 1062 > eval:80:21
    start speech.js:45
```

I am using Firefox 62.0.2 on Ubuntu

When I print an error I am getting
```
return error TypeError: "can't assign to property "onresult" on false: not an object"
    start speech.js:50
    start speech.js:35
    record Index.vue:96
    click Index.vue:45
    VueJS 4
        invokeWithErrorHandling
        invoker
        invokeWithErrorHandling
        $emit
    click QBtn.js:101
    VueJS 3
        invokeWithErrorHandling
        invoker
        _wrapper
Index.vue:107

TypeError: can't assign to property "lang" on false: not an objectapp.js line 1062 > eval:80:21
    start speech.js:45
```

**Comment by patrickmonteiro** [comment]
Hi @valasek ,

sorry for the delay, i am in the process of changing city and job, and i am without internet (only 4g of cell phone). Today I did a quick check and got the same error on windows firefox.
Looking at exactly where the error occurs, I went to see if SpeechRecognition is available for firefox, and the table shows that it still doesn't have compatibility. Then you will only be able to use SpeechSynthesisUtterance () in firefox.

Look at the Browser compatibility:

https://developer.mozilla.org/en-US/docs/Web/API/SpeechRecognition
